src/runtime/config.rs

```diff
@@ -48,9 +99,14 @@ pub trait ConfigureActor: Actor {
     fn with_stream<S, E>(self, stream: S) -> SpawnConfig<Self>
     where
         S: 'static + Stream<Item = E> + Unpin + Send,
-        E: 'static + DynEvent<Self> + Send + Sync,
+        E: 'static + DynEvent<Self>,
     {
         SpawnConfig::<Self>::new(self).with_stream(stream)
     }
+
+    /// Sets whether the actor's address should be added to the registry.
+    fn add_to_registry(self, add: bool) -> SpawnConfig<Self> {
```

Comment:
Idk but `with_registration(self, enable: bool)` might be nicer. Ignore if you disagree.